graylog2-web-interface/src/components/events/events/ColumnRenderers.tsx

```diff
@@ -40,7 +40,7 @@ const EventDefinitionRenderer = ({
   eventDefinitionId: string;
   meta: EventsAdditionalData;
 }) => {
-  const title = meta?.context?.event_definitions?.[eventDefinitionId]?.title;
+  const title = meta?.context?.event_definitions?.[eventDefinitionId]?.title || eventDefinitionId;
```

Comment:
We don't need this check since it is already part of the `EventDefinitionLink` component.